### ISSUE TITLE
Support for grails 3.x

### ISSUE THREAD

**Issue by imadi** [mathifonseca/grails-logentries#3]

**Comment by imadi** [comment]
I am working on it

**Comment by mathifonseca** [comment]
Hi @imadi, were you able to finish the migration? I'll publish the new version whenever you want.

**Comment by omadevor** [comment]
@imadi @mathifonseca Any updates on this issue?

**Comment by mathifonseca** [comment]
@omadevor never got a response 😢, do you want to create a PR?

**Comment by omadevor** [comment]
Do you know if @imadi did any work on it? If so I can continue from there and then make the PR.

**Comment by mathifonseca** [comment]
@omadevor I don't think so, it's been a while since the other comment. I would love to help you but I'm not using Grails 3, so I wouldn't know how to do it correctly.
